pkgs/os-specific/linux/alsa-firmware/default.nix

```diff
@@ -28,7 +25,7 @@ stdenv.mkDerivation rec {
   '';
 
   meta = {
-    homepage = http://www.alsa-project.org/main/index.php/Main_Page;
+    homepage = http://www.alsa-project.org/;
```

Comment:
Just making this one consistent with all the other alsa-project homepage meta attributes.